src/main/java/org/broadinstitute/hellbender/utils/report/GATKReportTable.java

```diff
@@ -664,4 +664,8 @@ public boolean equals(final GATKReportTable table) {
                 return underlyingData;
         }
     }
+
+    public String getTableDescription() {
```

Comment:
Please move this so its up next to the other getters, and add javadoc.

Reply:
done.  since we're adding javadoc to one getter, i added it to all the others (which didnt have it).  are you OK squashing when merging?

Reply:
done.  since we're adding javadoc to one getter, i added it to all the others (which didnt have it).  are you OK squashing when merging?